lib/board.ex

```diff
@@ -12,4 +12,53 @@ defmodule Board do
   def place_a_symbol(board, player_move, symbol) do
     Map.replace(board, String.to_integer(player_move), symbol)
   end
+
+  @spec has_player_won(map()) :: boolean()
+  def has_player_won(board) do
```

Comment:
```suggestion
  @spec has_player_won?(map()) :: boolean()
  def has_player_won?(board) do
```

For predicate functions I would recommend appending a `?` to the end of this function. That's a convention in most languages that support the `?` character in function names.

Reply:
I didn't know about this but I like it alot, Ill be sure to put this in